entrypoint.sh

```diff
@@ -20,6 +20,11 @@ if [ ! -d "/github/workspace/${INPUT_WIKI_FOLDER}" ]; then
 fi
 cp -a ${INPUT_WIKI_FOLDER}/. /wiki
 
+if [ "${INPUT_IGNORE_SAFE_WARNINGS}" = "true" ]; then
+    chown -R $USER:$USER /wiki
```

Comment:
I think you *should* only need one or the other (chown or safe.directory config)

Reply:
Let's see what happens without `chown` then

Reply:
Seems to work: https://github.com/GTNewHorizons/ServerUtilities/actions/runs/7379573039/job/20075988950